src/test/javascript/spec/component/complaints/complaints-for-tutor.component.spec.ts

```diff
@@ -13,6 +13,12 @@ import { ComplaintResponse } from 'app/entities/complaint-response.model';
 import { By } from '@angular/platform-browser';
 import { HttpResponse } from '@angular/common/http';
 import { of } from 'rxjs';
+import { Course } from 'app/entities/course.model';
+
+// Mock getCourseFromExercise(exercise) to get a course even if there isn't a course.
+jest.mock('app/entities/exercise.model', () => ({
```

Comment:
I think this would make more sense in the beforeEach block.

Reply:
Adding the mock to the beforeEach block causes the function not to be mocked properly and consequently the tests fail :thinking: So I think we should leave it as it is.